docs/notification-scripts.md

```diff
@@ -14,7 +14,10 @@ The interface for notification scripts is as follows:
   to finish, which will cause Matterhorn to delay looking at incoming
   network messages, etc.
 
-* Matterhorn will invoke the command with three arguments:
+* The notifier behavior changes according to the notifier version
+  specified in the configuration file.
```

Comment:
Can this mention the new `activityNotifyVersion` setting explicitly?

Reply:
Resolved by commit 111ba526 .

Reply:
Resolved by commit 111ba52 .